test/ui/FieldValueTest.ts

```diff
@@ -71,7 +71,23 @@ module Coveo {
         expect(test.cmp.element.textContent).toBe('this, is, sparta');
       });
 
-      it('separator should specify the string used to split a multi values field', () => {
+      it('viewSeparator should modify the string displayed between values of a multi value field', () => {
```

Comment:
Grammar nazi : I would use "multi-value" instead of "multi value" or "multi values" 😛